jha/src/views/enrolment/PersonalInfoPage.vue

```diff
@@ -343,9 +343,8 @@
                       && !$v.moveFromOrigin.required"
                     aria-live="assertive">Jurisdiction of origin is required.</div>
                   <div class="text-danger"
-                    v-if="$v.moveFromOrigin.$dirty
-                      && !$v.moveFromOrigin.nonCanadaValidator"
-                    aria-live="assertive">Jurisdiction of origin cannot be Canada.</div>
+                      v-if="moveFromOrigin === 'Canada'"
```

Comment:
I can see you removed the v-if that checks if the validation is dirty or not. Is that intentional? All of the rest seem to have it.

Reply:
See comment under Harry's requested changes